save-frontend/src/main/kotlin/com/saveourtool/save/frontend/externals/reactace/ReactAce.kt

```diff
@@ -64,5 +65,18 @@ external interface AceEditorProps : Props {
     // keyboardHandler?: string;
```

Comment:
Maybe delete unused parameters for now? And add them later as needed.

Reply:
Well, I wanted to make it easier to support `react-ace` features as the one who needs it won't go to GitHub trying to find all the possible props.